docs/hr/content/use_cases/classical_tasks/resnet_features.md

```diff
@@ -1,6 +1,5 @@
-# Image features
 
-# Building an Image Feature Database in `torchvision`
+# Building an Image Feature-Store Using Torchvision on MongoDB
 
 ```python
 !pip install superduperdb==0.0.12
```

Comment:
Obsolete version

Reply:
Thanks ,

This pr is mainly for the nesting, renaming and merging of pages. I have another ticket for diving into the content. There I will test the versions and the code as well